sentry-ruby/spec/sentry/hub_spec.rb

```diff
@@ -151,6 +151,12 @@
       end.to change { transport.events.count }.by(1)
     end
 
+    it "raises error when passing a non-exception object" do
+      expect do
+        subject.capture_exception("String")
+      end.to raise_error(ArgumentError)
```

Comment:
Maybe you could match the exception message too `raise_error(ArgumentError, "expect the argument")`?